2.x/Dockerfile

```diff
@@ -32,7 +29,7 @@ RUN set -x \
     && apk add --no-cache --virtual .run-deps \
         libstdc++ \
         readline \
-        libressl \
+        openssl \
```

Comment:
This is the only change I don't get. I don't mind, but it would be good to describe exact problem with libressl.

Reply:
Sorry, I don't have any idea why it happens.
If I use libressl, I get

```

/usr/src/tarantool/third_party/curl/lib/vtls/openssl.c: In function 'set_ssl_version_min_max':
/usr/src/tarantool/third_party/curl/lib/vtls/openssl.c:2195:9: error: implicit declaration of function 'SSL_CTX_set_min_proto_version' [-Werror=implicit-function-declaration]
     if(!SSL_CTX_set_min_proto_version(ctx, ossl_ssl_version_min)) {
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/src/tarantool/third_party/curl/lib/vtls/openssl.c:2231:7: error: implicit declaration of function 'SSL_CTX_set_max_proto_version' [-Werror=implicit-function-declaration]
   if(!SSL_CTX_set_max_proto_version(ctx, ossl_ssl_version_max)) {
       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
cc1: some warnings being treated as errors
make[5]: *** [Makefile:2187: vtls/libcurl_la-openssl.lo] Error 1
make[5]: *** Waiting for unfinished jobs....
make[5]: Leaving directory '/usr/src/tarantool/build/curl/work/lib'
make[4]: Leaving directory '/usr/src/tarantool/build/curl/work/lib'
make[4]: *** [Makefile:771: all] Error 2
make[3]: *** [Makefile:945: all-recursive] Error 1
make[3]: Leaving directory '/usr/src/tarantool/build/curl/work'
make[2]: *** [CMakeFiles/bundled-libcurl-project.dir/build.make:111: build/curl/work/stamp/bundled-libcurl-project-build] Error 2
make[2]: Leaving directory '/usr/src/tarantool'
make[1]: Leaving directory '/usr/src/tarantool'
make[1]: *** [CMakeFiles/Makefile2:420: CMakeFiles/bundled-libcurl-project.dir/all] Error 2
make: *** [Makefile:150: all] Error 2
```

The root of the problem is in curl

Reply:
Okay so.